visuals.Rmd

```diff
@@ -227,6 +227,103 @@ vis_drake_graph(
 )
 ```
 
+## Node Selection
+
+*(Supported in drake > 7.7.0, non-inclusive)*
```

Comment:
Another thing I forgot to mention: could we say "Supported in drake > 7.7.0 only?" I believe non-inclusive is implied in the strict inequality, and the emphasis is the lack of support in earlier versions.